Makefile

```diff
@@ -91,6 +91,9 @@ endif
 ifdef UPM_CI
 test-suite:
 	go get gotest.tools/gotestsum
+	venv="$$(mktemp -d)"; \
+	python -m venv "$$venv"; \
+	source "$$venv/bin/activate"; \
```

Comment:
is there a different way to do this? i'd rather be able to support multiple venvs being used in a test suite run

Reply:
Yeah -- this was what motivated my comment over Slack about before/after's -- I don't like this either, especially since it pollutes the environment between test cases. I just wanted to see ✅ at least once before delving back into the ❌'s

Reply:
gotcha. can you add a TODO to move our command invocation to being inside of a shell, where we can define before/afters?